sources/mongodb/README.md

```diff
@@ -0,0 +1,54 @@
+# Mongo Database
+Mongo is a document-oriented database that store json-like documents is one of the most used NoSQL databases. The Mongo Database `dlt` is a verified source and pipeline example that makes it easy to load data from your Mongo database to a destination of your choice. It offers flexibility in terms of loading either the entire database or specific collections to the target.
+
+## Initialize the pipeline with Mongo Database verified source
+```bash
+dlt init mongodb bigquery
+```
+Here, we chose BigQuery as the destination. Alternatively, you can also choose redshift, duckdb, or any of the other [destinations.](https://dlthub.com/docs/dlt-ecosystem/destinations/)
+
+## Setup verified source
+
+To setup the Mongo Database Verified Source read the [full documentation here.](https://dlthub.com/docs/dlt-ecosystem/verified-sources/mongodb)
+
+## Add credentials
+1. Open `.dlt/secrets.toml`.
+2. In order to continue, you can use the a connection URL and database that looks like this:
+    ```bash
+    connection_url = "mongodb://dbuser:passwd@host.or.ip:27017"
+    database = "local"
```

Comment:
explain that database is optional and if one is provided in connection string - that one will be used